### DISCUSSION TITLE
Insights - Graphs using black line when dark mode on

### DISCUSSION THREAD

**Discussion by rafamarts** [community/community#38302] [Projects and Issues]
In GitHub Projects, using Insights, there is a black line in graph that we can see very well.

In these image, there is a sample of black line that should show the Backlog.

Are there  a way to change this color manually?

![Sample](https://user-images.githubusercontent.com/5350121/200046426-d8747f72-9fba-4479-8e23-64f5aadd1ffa.png)

**Comment by david-engelmann** [comment]
[Similar discussion on the issue](https://github.com/community/community/discussions/38184)

**Reply by rafamarts** [reply]
Thank you!! Sorry by duplicate it.

**Reply by david-engelmann** [reply]
Not a problem at all, figured I'd tie the two
